Modules/System/Time Zone Management/Permissions/TimeZoneMgtObjects.PermissionSet.al

```diff
@@ -0,0 +1,10 @@
+permissionset 8720 TimeZoneMgtObjects
```

Comment:
This permission set should probably be called "Time Zone Management - Objects", just like the caption. This one should be internal. You could then create a new permission set, which isn't internal and has the name "Time Zone Management - Execute". For now, that permission set would only contain the objects one...

Reply:
@JesperSchulz  Keep in mind, that PermissionSets should have a max length of 20 chars.

Reply:
30, I believe? We've got plenty with a length of more than 20 chars (e.g. permissionset 9010 "AAD User Management - Objects").

Reply:
I recieve the following warning in VS Code.
![image](https://user-images.githubusercontent.com/56249171/210389524-04424fa2-f03b-4061-b92a-44f3248dfec7.png)
Maybe this warning is outdated.
But there has been some limitation.
At least the Role ID in`table 2000000167 "Aggregate Permission Set"` is limited to Code[20]
and the Name is limited to Text[30].
Therefore we always add a maxlength = 30 to the PermissionSet Caption

```al
table 2000000167 "Aggregate Permission Set"
{
    DataPerCompany = false;
    Scope = Cloud;

    //WriteProtected=True;
    fields
    {
        field(1; Scope; Option)
        {
            OptionMembers = System,Tenant;
        }
        field(2; "App ID"; Guid)
        {
        }
        field(3; "Role ID"; Code[20])
        {
        }
        field(4; Name; Text[30])
        {
        }
        field(5; "App Name"; Text[250])
        {
        }
    }
    keys
    {
        key(pk; Scope, "App ID", "Role ID")
        {
        }
    }

    fieldgroups
    {
        fieldgroup("DropDown"; "Role ID", Name, "App Name")
        {

        }
    }
}
```

Reply:
My compiler keeps crashing on the app.json (accessviolation error) so my code analyzer breaks like mad, but I have updated the name.

Reply:
About permission sets... from 2023 wave 1, we're going to implement inherent permissions on these kinds of modules (permissions defined in the objects, which aren't controllable). Nobody wants to control permissions on "Time Zone" calculations - everyone should just be able to use that code. Hence, please do put it into "System Application - Objects", and I will convert the permissions to inherent permissions in the objects, when porting the code to 2023 Wave 1.

Reply:
Makes sense. I have added it to "System Application - Objects".

Reply:
Permissions added to System Application - Objects